cli/index.cli.js

```diff
@@ -31,16 +32,20 @@ if (!program.entry || !program.dir) {
   process.exit();
 }
 
+
+const config = require('codecrumbs.config.json');
```

Comment:
@idoo please use `codecrumbs.config.js`, not `json`

Reply:
fixed